service/controller/resource/monitoring/pvcresizingresource/create.go

```diff
@@ -34,6 +37,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			desiredPVCSize := cluster.GetAnnotations()[key.PrometheusVolumeSizeAnnotation]
 			desiredVolumeSize := resource.MustParse(pvcresizing.PrometheusVolumeSize(desiredPVCSize))
 
+			needToUpdatePrometheis := false
```

Comment:
```suggestion
			needToUpdatePrometheus := false
```

Reply:
Herve learned me that the plurial of prometheus is prometheis :-)